server/manager/src/main/java/org/apache/accumulo/manager/tableOps/compact/CompactionDriver.java

```diff
@@ -186,6 +186,7 @@ public int updateAndCheckTablets(Manager manager, FateId fateId)
         var tabletsMutator = ample.conditionallyMutateTablets(resultConsumer)) {
 
       CompactionConfig config = CompactionConfigStorage.getConfig(manager.getContext(), fateId);
+      var time = manager.getSteadyTime();
```

Comment:
May be better to grab the steady time each time its needed.  Thinking for a large amount of tablets it could take a while to scan and write out data and this may get stale.  Seems like it ok to have this be different for different tablets.  Looking at the impl of manager.getSteadyTime(), it seems like it should be quick to call.